tests/test_view_stream.py

```diff
@@ -121,6 +122,43 @@ def test_write_only(self):
         self.stream.clear()
         self.assertEqual(self.stream.view.size(), 0)
 
+    def _compare_print(self, *args, **kwargs):
+        self.stream.clear()
+        self.stream.print(*args, **kwargs)
+
+        s = StringIO()
+        print(*args, file=s, **kwargs)
+
+        self.assertContents(s.getvalue())
+
+    def test_print(self):
+        text = "Hello, World!"
+        number = 42
+
+        self._compare_print(text, number)
+        self._compare_print(text, number, sep=',', end=';')
+        self._compare_print(text, number, sep='', end='')
+
+    def test_print_no_indent(self):
+        text = "    "
+
+        self.stream.view.settings().set('auto_indent', True)
+        self.stream.print(text)
+        self.assertContents(text + "\n")
+
+    def test_print_read_only(self):
+        self.stream.view.set_read_only(True)
+
+        self.assertRaises(ValueError, self.stream.print, 'foo')
+        self.assertRaises(ValueError, self.stream.clear)
+
+        self.stream.force_writes = True
```

Comment:
I believe this belongs to a separate test case.

Reply:
If so, it should be changed in `test_write_read_only` as well.

Reply:
Yes, probably. That test case wasn't included in the diff.